### PR TITLE
Rpethkar

### PR DESCRIPTION
**Pull request by Rajashree-Pethkar**
Homework 2:

1. Completed Exercise 01 - Flexbox Froggy
2. Completed Exercise 02 - Grid Garden
3. Completed Exercise 03 - Count
    - Set the background color as per the homework
    - Added bootstrap styling
    - Implemented the highlight functionality
4. Completed Exercise 04 - Background
    - Implemented color changes functionality
    - Added the event listeners
    - Added bootstrap styling
    - Updated the html code as per homework
5. Completed Exercise 05 - Form
    - Added bootstrap styling
    - Updated the html as per the homework

### CODE REVIEW

File: hw2/03-count.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const input = document.getElementById('input');
+
+function handleKeyUp(event) {
+  const inputText = event.target.value;
+
+  const mainDiv = document.getElementById("text");
+  const textVal = mainDiv.innerText;
+  if (inputText.length > 0) {
+    // Found the regex from internet and with the help from a friend, I do not have the url.
+    let regexp = new RegExp(`\\b(${inputText})\\b`, "ig");
```
Comment (Pranav-PSU) [inline hw2/03-count.js]: Const instead of let can be used because regexp is not getting reassigned

File: hw2/03-count.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  }
+}
+
+input.addEventListener('keyup', handleKeyUp);
+
+
+// References
+// https://stackoverflow.com/questions/52743841/find-and-highlight-word-in-text-using-js
+// https://stackoverflow.com/questions/32130130/how-to-highlight-all-occurrences-of-a-word-on-a-page-with-javascript-or-jquery
+
```
Comment (Pranav-PSU) [inline hw2/03-count.js:29]: Consider formatting this file with prettier

File: hw2/05-form.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        <div class="mb-3">
+          <label for="username" class="form-label h6"><strong>Full Name:</strong></label>
+          <input type="text" id="username" name="username" class="form-control" />
+        </div>
+        <div class="mb-3">
+          <label for="email" class="form-label h6"><strong>Email:</strong></label>
+          <input id="email" name="email" type="email" class="form-control" />
+        </div>
+        <div class="mb-3">
+          <label for="registrationStatus" class="form-label h6"><strong>Registration Status:</strong></label>
```
Comment (Pranav-PSU) [inline hw2/05-form.html:26]: The value of the for attribute of the label element must be the ID of a form control.

**Review by Pranav-PSU — CHANGES_REQUESTED** [review]
Good job with the homework. A couple of minor changes are suggested. Otherwise, everything looks good.

**Review by rohan-singh1 — APPROVED** [review]
Issues have already been pointed out and addressed. Looks good.

**Review by Pranav-PSU — APPROVED** [review]

**Pull request merged**